translations/translation_de.xml

```diff
@@ -226,12 +226,12 @@
 		<text name="COURSEPLAY_HEADLAND_CORNER_TYPE_SMOOTH"  				text="weich" />
 		<text name="COURSEPLAY_HEADLAND_CORNER_TYPE_SHARP"  				text="scharf"/>
 		<text name="COURSEPLAY_HEADLAND_CORNER_TYPE_ROUND"  				text="rund" />
-		<text name="COURSEPLAY_CENTER_MODE"									text="Feldmitte (experimental)" />
+		<text name="COURSEPLAY_CENTER_MODE"									text="Feldmitte" />
 		<text name="COURSEPLAY_CENTER_MODE_UP_DOWN"							text="hoch/runter" />
 		<text name="COURSEPLAY_CENTER_MODE_CIRCULAR"						text="kreisförmig" />
 		<text name="COURSEPLAY_CENTER_MODE_SPIRAL"						    text="spiralförmig" />
-		<text name="COURSEPLAY_CENTER_MODE_LANDS"						    text="Lands" />
-		<text name="COURSEPLAY_NUMBER_OF_ROWS_PER_LAND"					    text="Number of rows per land" />
+		<text name="COURSEPLAY_CENTER_MODE_LANDS"						    text="Beete (innen nach aussen)" />
+		<text name="COURSEPLAY_NUMBER_OF_ROWS_PER_LAND"					    text="Anzahl Bahnen pro Beet" />
```

Comment:
Bahnen pro 'Insel' ? Ohne Anzahl.

Reply:
mit dem Wörtchen "Anzahl" war ich mir tatsächlich selber unschlüssig ;) Ich denk mal in beiden Fällen weiß jeder was gemeint ist. Da im zugehörigen Dropdown-Feld sowieso immer eine Zahl steht, kann man das Wort auch genausogut weglassen

Reply:
Ja gut lassen wir das erst mal so...
Aber mit Beete können halt die wenigsten etwas anfangen.... naja mal sehen...